frontend/catalyst/jit.py

```diff
@@ -718,6 +725,14 @@ def _validate_configuration(self):
             )
 
     def _verify_static_argnums(self, args):
+        if not (
+            isinstance(self.compile_options.static_argnums, tuple)
+            and all(isinstance(arg, int) for arg in self.compile_options.static_argnums)
+        ):
```

Comment:
While this can be beneficial sometimes, I have to say I find the first form much easier to read in this case, you can pretty much directly read off the conditions one by one, while for the second form you have to jump around parsing the variables and putting them in your head context while looking a the condition.

Reply:
Yeah, happy to leave it up to @mehrdad2m . :)

Reply:
I changed it a bit to make it nicer.